jaeger_client/span.py

```diff
@@ -81,11 +81,27 @@ def set_tag(self, key, value):
             if key == ext_tags.SAMPLING_PRIORITY and not self._set_sampling_priority(value):
                 return self
             if self.is_sampled():
-                tag = thrift.make_string_tag(
-                    key=key,
-                    value=value,
-                    max_length=self.tracer.max_tag_value_length,
-                )
```

Comment:
Thanks for the PR.

Nit: I think this is not the only place. I would rather create a single method thrift.create_tag and do the dispatching by type inside it, to new private methods.

Reply:
sure I can try and do that